chapters/ar/chapter1/1.mdx

```diff
@@ -0,0 +1,58 @@
+
+<div dir="rtl" style="direction:rtl;text-align:right;">
+
+# مقدمة
+
+## مرحبا بكم في دورة 🤗!
+
+<Youtube id="00GKzGyWFEs" />
+
+ستعلمك هذه الدورة التدريبية حول معالجة اللغة الطبيعية (NLP) باستخدام مكتبات من نظام [Hugging Face🤗] (https://huggingface.co) - [Transformers🤗] (https://github.com/huggingface/transformers) و [Datasets🤗] (https://github.com/huggingface/datasets) و [Tokenizer🤗] (https://github.com/huggingface/tokenizers) و [Accelerate🤗] (https://github.com/huggingface/accelerate) وكذلك [Hugging Face Hub] (https://huggingface.co/models). و هذا كله مجاني تمامًا وبدون إعلانات.
+
+
+## ماذا تتوقع؟
+
+فيما يلي نظرة عامة موجزة عن الدورة:
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface-course/documentation-images/resolve/main/en/chapter1/summary.svg" alt="Brief overview of the chapters of the course.">
+
+- تقدم الفصول من 1 إلى 4 مقدمة للمفاهيم الرئيسية لمكتبة Transformers🤗. بنهاية هذا الجزء من الدورة التدريبية ، ستكون على دراية بكيفية عمل نماذج المحولات Transformer وستعرف كيفية استخدام نموذج من [Hugging Face Hub] (https://huggingface.co/models) ، تعديله علي مجموعة بيانات محددة ومشاركة نتائجك على Hub!
+- الفصول من 5 إلى 8 تعلم أساسيات مكتبة مجموعات البيانات Datasets🤗و كذلك مكتبة الترميز Tokenizer🤗 قبل الغوص في مهام NLP الكلاسيكية. بنهاية هذا الجزء ، ستكون قادرًا على معالجة مشكلات NLP الأكثر شيوعًا بنفسك.
+- الفصول من 9 إلى 12 تتجاوز NLP ، وتستكشف كيف يمكن استخدام نماذج المحولات لمعالجة مهام حاسوبية أخري كمعالجة الكلام  والرؤية الحاسوبية. ستتعلم أيضا كيفية إنشاء ومشاركة العروض التوضيحية لنماذجك ، وتحسينها لبيئات الإنتاج. بنهاية هذا الجزء ، ستكون جاهزًا لتطبيق المحولات (تقريبًا) على أي مشكلة في التعلم الآلي!
+
+هذه الدورة:
+
+* تتطلب معرفة جيدة بلغة البرمجة Python
+* من الأفضل أخذ هذه الدورة بعد دورة تعليمية تمهيدية ، مثل [fast.ai's] (https://www.fast.ai) [التعلم العميق العملي للمبرمجين] (https://course.fast.ai) أو أحد البرامج التي طورتها [DeepLearning.AI] (https://www.deeplearning.ai)
+* لا تتوقع معرفة مسبقة بـ [PyTorch] (https://pytorch.org) أو [TensorFlow] (https://www.tensorflow.org) ، على الرغم من أن الإلمام بأيٍّ منهما سيساعد
+
+بعد إكمال هذه الدورة ، نوصي بالاطلاع على DeepLearning.AI [تخصص معالجة اللغة الطبيعية (NLP)] (https://www.coursera.org/specializations/natural-language-processing؟utm_source=deeplearning-ai&utm_medium=institutions&utm_campaign=20211011- nlp-2-hugging_face-page-nlp-Refresh) ، والتي تغطي مجموعة واسعة من نماذج البرمجة NLP مثل Naive Bayes و LSTMs التي تستحق المعرفة عنها!
+
+## من نحن؟
+
+عن المؤلفين:
+
+**Matthew Carrigan** مهندس تعلم الآلة في Hugging Face. يعيش في دبلن بأيرلندا وعمل سابقًا كمهندس ML في Parse.ly وقبل ذلك كان باحثًا بعد الدكتوراه في Trinity College Dublin. إنه لا يعتقد أننا سنصل إلى الذكاء الاصطناعي العام من خلال توسيع البيئات الحالية ، ولكن لديه آمال كبيرة في خلود الروبوت بغض النظر عن ذلك.
```

Comment:
I suggest wrapping paragraphs that start with EN words with 
```
<div style="direction:rtl;">
</div>
```
because these sentences are hard to read backward->forward->backward etc.

Reply:
Thanks for suggesting this @giyaseddin! @HishamKhdair if you need some additional guidance, I suggest checking out how the Farsi files were translated in the `chapters/fa` folder

Reply:
Thanks for the comments @giyaseddin and @lewtun . 

You might be not able to read it only inside .md file but the rendered page works very well. The whole content is wrapped with `<div dir="rtl">` , and all sounds alright. 

I checked and verified this with "Markdown Preview Enhanced" in vscode in addition to other markdown live preview tools. All seems to be correct and intact.

Reply:
I noticed this by checking [this link](https://github.com/huggingface/course/blob/9834902cbe635fefeeaff9e6736c9acbc0d89bb7/chapters/ar/chapter1/1.mdx), direction confusion -especially code snippets- which might as you said not be the actual production rendering. Can't wait to see the final result.